solidity/contracts/utility/ContractRegistry.sol

```diff
@@ -15,6 +15,8 @@ import './interfaces/IContractRegistry.sol';
     Note that contract names are limited to 32 bytes UTF8 encoded ASCII strings to optimize gas costs
 */
 contract ContractRegistry is IContractRegistry, Owned, Utils {
+    bytes32 public constant CONTRACT_REGISTRY = "ContractRegistry";
```

Comment:
currently the ContractRegistry doesn't have access to ContractIds (do we want to add it?). The reason I hardcoded it into the ContractRegistry is so that after construction the registry is already pointing to itself.